tools/dbinitializer/main.go

```diff
@@ -122,10 +122,31 @@ func Initialize(
 }
 
 func initSysConfig(svrConf *contractAddr, bscTestNetworkRPC, localTestNetworkRPC string) []*sysconfig.SysConfig {
+
+	// to config gas for different transaction types, need to be evaluated and tune these values
+	gasFeeConfig := make(map[uint32]map[int]int64) // asset id -> (tx type -> gas fee value)
+	gasFeeConfig[0] = make(map[int]int64)          // bnb asset
+	gasFeeConfig[0][types.TxTypeTransfer] = 10000000000000
```

Comment:
can we new initiate a map first and then assign it to `gasFeeConfig`, and can we avoid using the magic `0`.

Reply:
sure, updated